packages/client/lib/MetadataClient/IssuanceInitiationMetadataClient.ts

```diff
@@ -0,0 +1,22 @@
+import { EndpointMetadata, IssuanceInitiationRequestPayload, IssuanceInitiationWithBaseUrl } from '@sphereon/openid4vci-common';
```

Comment:
Is there a good reason to split this up already? Why not simply accept both versions? It only is a name change at this point.

Reply:
reverted